src/test/java/com/egov/core-services/requestPayload/user/update/updateUser.json

```diff
@@ -0,0 +1,53 @@
+{
+    "RequestInfo": {
+        "authToken": "#(authToken)"
+    },
+    "user": {
+        "id": 4657,
+        "userName": "EMP-1013-000335",
+        "salutation": null,
+        "name": "AUTOMATIONUSER",
+        "gender": "MALE",
+        "mobileNumber": "9999999999",
+        "emailId": "test@email.com",
+        "altContactNumber": null,
+        "pan": null,
+        "aadhaarNumber": null,
+        "permanentAddress": null,
+        "permanentCity": null,
+        "permanentPinCode": null,
+        "correspondenceAddress": "TEST ADDRESS",
+        "correspondenceCity": "jalandar",
+        "correspondencePinCode": null,
+        "active": true,
+        "locale": null,
+        "type": "EMPLOYEE",
+        "accountLocked": false,
+        "accountLockedDate": 0,
+        "fatherOrHusbandName": "TEST",
+        "signature": null,
+        "bloodGroup": null,
+        "photo": null,
+        "identificationMark": null,
+        "createdBy": 435,
+        "lastModifiedBy": 1,
+        "tenantId": "pb.jalandhar",
+        "roles": [
+            {
+                "code": "STADMIN",
+                "name": "State Administrator",
+                "tenantId": "pb.jalandhar"
+            },
+            {
+                "code": "SUPERUSER",
+                "name": "Super User",
+                "tenantId": "pb.jalandhar"
+            }
+        ],
+        "uuid": "b707e0ff-647f-4524-b2f2-677c26295987",
+        "createdDate": "07-01-2021 19:32:06",
+        "lastModifiedDate": "07-01-2021 20:06:46",
+        "dob": "1/1/1988",
+        "pwdExpiryDate": "07-04-2021 19:32:06"
```

Comment:
these values should be stored in yaml file

Reply:
I agree. Kindly do the same